securedrop/journalist_templates/_sources_confirmation_final_modal.html

```diff
@@ -2,19 +2,19 @@
   <a href="#close" class="external" aria-label="{{ gettext('Close') }}"></a>
   <dialog open id="delete-confirm-menu-dialog" aria-labelledby="confirm-heading">
     <h2 id="confirm-heading" class="paragraph-spacing">
-      {{ gettext('When the account for a source is deleted:') }}
+      {{ gettext('When the account for a <abr title="user">source</abr>is deleted:') }}
     </h2>
     <ul>
-      <li>{{ gettext('The source will no longer be able to log in with their codename.') }}</li>
+      <li>{{ gettext('The <abr title="user">source</abr> will no longer be able to log in with their <abbr title="user name">codename</abbr>.') }}</li>
```

Comment:
It's unclear that terms like `source` and `codename` are ones that should have an abbr tag (vs the OATH-HOTP instance below, which definitely should!) - is there a reason they're included?

Reply:
The reason why I thought it would be nice to have was because its a bit unclear as a new user what a codename is as its a term used specifically by the app. Let me know if I need to update @zenmonkeykstop